account/forms.py

```diff
@@ -43,3 +43,36 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = Account
         fields = ('email', 'username', 'password1', 'password2')
+
+
+class LoginForm(AuthenticationForm):
+    """
+    ログインフォーム
+    """
+    username = forms.CharField(
+        required=True,
+        label=_('username'),
+        error_messages={'required': 'ユーザー名を入れて下さい．'},
+        validators=[MinLengthValidator(1),
+            MaxLengthValidator(30)])
+    password = forms.CharField(
+        required=True,
+        label=_("password"),
+        error_messages={'required': 'パスワードを入れて下さい．'},
+        widget=forms.PasswordInput())
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+
+class ProfileForm(forms.ModelForm):
+    """
+    プロフィールフォーム
+    """
+    profile = forms.CharField(label='プロフィール', widget=forms.Textarea)
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    class Meta:
+        model = Account
+        fields = {'profile'}
```

Comment:
ModelFormを継承しているのであれば、Metaクラスを活用してあげた以下のような書き方が自然かなと思います。
forms.Textareaに引数としていくつか指定してあげることで、formの大きさなどのカスタマイズも可能ですので、余力があればやってみても良いと思います。
```suggestion
class ProfileForm(forms.ModelForm):
    """
    プロフィールフォーム
    """
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)

    class Meta:
        model = Account
        fields = {'profile'}
        widgets = {
            'profile': forms.Textarea
        }
        labels = {
            'profile': 'プロフィール'
        }
```